engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/tasks-nutupane.factory.js

```diff
@@ -10,87 +10,31 @@
  *   source of data instead of the standard `query` method. This
  *   effectively creates Nutupane table that is updated by polling.
  */
-angular.module('Bastion.tasks').factory('TasksNutupane',
-    ['Task', 'Nutupane', function (Task, Nutupane) {
-        var TasksNutupane = function () {
-            var self = this;
-
-            Nutupane.call(self, Task, {});
-
-            self.existingTasks = {};
-
-            self.registerSearch = function (params) {
-                if (!self.searchId) {
-                    self.searchId = Task.registerSearch(params, self.updateTasks);
-                }
-            };
-
-            self.unregisterSearch = function () {
-                if (self.searchId) {
-                    Task.unregisterSearch(self.searchId);
-                    self.searchId = undefined;
-                }
-            };
-
-            self.load = function () {
-                self.table.working = true;
-            };
-
-            self.updateTasks = function (tasks) {
-                self.refreshTasks(tasks);
-                self.deleteOldRows(tasks);
-                self.table.working = false;
-                self.table.refreshing = false;
-            };
-
-            self.table.search = function () {
-                self.table.resource.offset = 0;
-                self.table.rows = [];
-                if (!self.table.working) {
-                    self.load();
-                }
-            };
-
-            // Updates values for existing tasks and adds new rows
-            self.refreshTasks = function (tasks) {
-                // we reverse because we add new items to the top of
-                // the table
-                _.each(tasks.reverse(), function (task) {
-                    var existingTask = self.existingTasks[task.id];
-                    if (existingTask) {
-                        _.each(task, function (value, key) {
-                            existingTask[key] = value;
-                        });
-                    } else {
-                        self.table.rows.unshift(task);
-                        self.existingTasks[task.id] = task;
-                    }
-                });
+angular.module('Bastion.tasks').factory('TasksNutupane', [
+    '$timeout',
+    'Task',
+    'Nutupane',
+    function($timeout, Task, Nutupane) {
+        var TasksNutupane = function() {
+            var tasksNutupane = new Nutupane(Task, {}, 'poll', {
+                // disableAutoLoad: true
+            });
```

Comment:
Should we just remove the empty `{}` last parameter?

Reply:
Yup - gone